modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/registry/AppFacRemoteRegistryAccessService.java

```diff
@@ -531,8 +545,135 @@ private AppFactoryResource convertResourceObject( AppFacResource appFacResource)
     	
     }
 
+    /**
+     * Create runtime properties in Kubernetes for applcations
+     *
+     * @param applicationId id of the application
+     * @param stage         stage of the application
+     * @param key           property key
+     * @param value         property value
+     * @param isSecured     property is secured or not
+     * @param version       version of the application
+     */
+    @Override
+    public void addRuntimeProperty(String applicationKey, String version, String stage, String key,
+            String value, boolean isSecured) throws AppFactoryException {
+        String tenantDomain = PrivilegedCarbonContext.getCurrentContext().getTenantDomain();
+
+        if (log.isDebugEnabled()) {
+            log.debug("Adding runtime property for application : " + applicationKey + " and version : " + version
+                    + " in stage " + stage + " with property : " + key + " in tenant domain : " + tenantDomain);
+        }
+
+        RuntimeProperty runtimeProperty = new RuntimeProperty();
+        Map<String, String> properties = new HashMap<String, String>();
+
+        boolean isChecked = isSecured;
```

Comment:
This is fixed in following pull request.
https://github.com/wso2/product-af/pull/800/commits/251406ca0505bcb4e46024e283b181a333ea49be